### ISSUE TITLE
Shader displays pink when lighting is toggled off

### ISSUE THREAD

**Issue by LtdJorge** [guycalledfrank/bakery-hdrp-shaders#2]
The shaders don't work when the option to override scene light and use a camera attached light is used (the light bulb symbol near Draw mode and 2D view toggle).
The compiler was complaining about **_DebugViewMaterial** not being defined, so I swapped the code with what seems to be a new version of the **ShaderPassForward.hlsl** and it cleared the error, but the materials are still pink.
I can make a PR if you want with the change, but I don't know if more things changed. There are no errors but some warnings appear.

**Comment by guycalledfrank** [comment]
Seems to work for me on HDRP 5.16.0 which is the latest available version via package manager. Newer versions change too frequently and most users will update via the manager.

**Comment by LtdJorge** [comment]
Yep, that's why I only opened an issue, it's not that important and things break fast. 6.7 is available via package manager though, but you need a newer Unity version. I was seeing the error in the latest alpha, 2019.3.a5, so it doesn't mean much.
